### PR TITLE
Add a bug report template.

### PR DESCRIPTION
**Pull request by zainab-ali**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+name: 🐞Bug report
+about: Help us improve by reporting a bug.
+labels: bug
+---
```
Comment (majk-p) [inline .github/ISSUE_TEMPLATE/bug.md:5]: We can also add the `[Bug]: ` prefix to the title
```suggestion
name: 🐞Bug report
about: Help us improve by reporting a bug.
labels: bug
title: "[Bug]: "
---
```

Comment (majk-p) [inline .github/ISSUE_TEMPLATE/bug.md:5]: WDYT about auto-assigning to us for a triage as well?
```suggestion
name: 🐞Bug report
about: Help us improve by reporting a bug.
labels: bug
assignees:
  - siriusxm/snapshot4s-maintainers 
---
```

File: .github/ISSUE_TEMPLATE/bug.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+Thank you for reporting a bug with snapshot4s! We appreciate your time and patience.
+
+# The problem
+
+<-- Describe the problem. -->
+
```
Comment (majk-p) [inline .github/ISSUE_TEMPLATE/bug.md:12]: I think we can also ask for things like snapshot4s version, sbt version, scala version, platform, test framework and if the they use cross version/project matrix

Comment (zainab-ali) [reply .github/ISSUE_TEMPLATE/bug.md:15]: That would definitely be useful.

**Review by majk-p — APPROVED** [review]

**Pull request merged**